packages/@lwc/integration-karma/test/polyfills/aria-properties/index.spec.js

```diff
@@ -114,17 +122,32 @@ function testAriaProperty(property, attribute) {
         const isNative = Object.getOwnPropertyDescriptor(Element.prototype, property)
             .set.toString()
             .includes('[native code]');
-        const settingUndefinedRemoves = () => {
-            // This test is just in case Chromium/WebKit change their behavior, or Firefox ships their version
+
+        // This test is just in case Chromium/WebKit/Firefox change their behavior
+        const settingValueRemoves = (val) => {
             const div = document.createElement('div');
-            div[property] = undefined;
+            div[property] = val;
             return div[property] === null;
         };
-        if (isNative && settingUndefinedRemoves()) {
-            // Native Webkit/Chromium – setting undefined is treated the same as null
-            falsyValuesThatRemove.push(undefined);
+
+        if (isNative) {
+            if (settingValueRemoves(undefined)) {
+                // Native Webkit/Chromium – setting undefined is treated the same as null
+                falsyValuesThatRemove.push(undefined);
+            } else {
+                falsyValuesThatDoNotRemove.push(undefined);
+            }
+            if (settingValueRemoves(null)) {
+                // As of this writing, Firefox is inconsistent with Chromium/WebKit and treats setting undefined/null
+                // as setting a string value: https://bugzilla.mozilla.org/show_bug.cgi?id=1853209
+                falsyValuesThatRemove.push(null);
+            } else {
+                falsyValuesThatDoNotRemove.push(null);
+            }
         } else {
-            // Our polyfill or the current spec – setting undefined is not treated like null
+            // Our polyfill - null removes
+            falsyValuesThatRemove.push(null);
+            // Our polyfill – setting undefined is not treated like null
             falsyValuesThatDoNotRemove.push(undefined);
         }
```

Comment:
A lot of these tests are kind of dumb – testing a thing, and then asserting that your test does the same thing it just did. Maybe once the browsers are all in alignment, we can vastly simplify this test.